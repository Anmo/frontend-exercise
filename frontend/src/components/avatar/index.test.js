import { render } from '@testing-library/react';
import { useRecoilValue } from 'recoil';

import UserAvatar from './index';

jest.mock('recoil', () => ({
	...jest.requireActual('recoil'),
	useRecoilValue: jest.fn()
}));

test('renders the normal UserAvatar component', () => {
	useRecoilValue.mockReturnValue('test');

	const { container } = render(<UserAvatar />);

	expect(container.firstChild).toMatchSnapshot();
});
