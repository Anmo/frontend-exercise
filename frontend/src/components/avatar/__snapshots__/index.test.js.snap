// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the normal UserAvatar 1`] = `
<button
  class="MuiButtonBase-root MuiIconButton-root"
  tabindex="0"
  type="button"
>
  <span
    class="MuiIconButton-label"
  >
    <div
      class="MuiAvatar-root MuiAvatar-circle"
    >
      <img
        class="MuiAvatar-img"
        src="https://www.gravatar.com/avatar/1aedb8d9dc4751e229a335e371db8058"
      />
    </div>
  </span>
  <span
    class="MuiTouchRipple-root"
  />
</button>
`;
